### DISCUSSION TITLE
I'm using sfdx-git-delta for my SFDX deployments using Bitbucket pipeline however cannot find absolute metadata which is being generated for deployment

### DISCUSSION THREAD

**Discussion by akshay-kumar12** [community/community#53006] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm using sfdx-git-delta for my SFDX deployments using Bitbucket pipeline however cannot find absolute metadata which is being generated for deployment.

my worry is we need more visibility on what the sgd plugin - the sfdx sgd:source:delta --to HEAD --from HEAD^ command is producing as metadata to be deployed. with all the different branches and the use of the sgd plugin (which currently does not give transparency of the deployment content)

Please help how can i achieve it within my Bitbucket pipeline.

Using below script..
script:
                - sfdx --version
                - sfdx force:auth:jwt:grant  --clientid $UAT_CONSUMER_KEY --jwtkeyfile build/UATserver.key --username $UAT_USERNAME --instanceurl https://test.salesforce.com
                - sfdx sgd:source:delta --to HEAD --from HEAD^ --output .
                - sfdx force:source:deploy --checkonly -u $UAT_USERNAME -x package/package.xml --testlevel RunSpecifiedTests -r AnyTest

**Comment by marlonrodler** [comment]
You need to install the plugin, place the following line before executing its command:
`echo 'y' | sfdx plugins:install sfdx-git-delta`
